bin/templates/cordova/lib/check_reqs.js

```diff
@@ -355,15 +355,16 @@ module.exports.check_android_target = function (originalError) {
 // Returns a promise.
 module.exports.run = function () {
     return Q.all([this.check_java(), this.check_android()]).then(function (values) {
-        console.log('ANDROID_HOME=' + process.env['ANDROID_HOME']);
-        console.log('JAVA_HOME=' + process.env['JAVA_HOME']);
+        console.log('Checking Java JDK and Android SDK versions');
 
         if (!String(values[0]).startsWith('1.8.')) {
-            throw new CordovaError(`Requirements check failed for JDK 8 ('1.8.*')`);
+            throw new CordovaError(
+              `Requirements check failed for JDK 8 ('1.8.*')! Detected version: ${values[0]}\nCheck your JAVA_HOME / PATH environment variables.`
```

Comment:
Build failed due to eslint error. I think you need to indent this line 2 more spaces to make eslint happy, for some reason.

I would also favor splitting the message into multiple lines, like I we did here (among some other places): <https://github.com/apache/cordova-android/blob/d9c08f12a7bceb47ee97ca8ca0e75548e78f999d/bin/templates/cordova/lib/check_reqs.js#L205-L208>

Reply:
The eslint error is now fixed.